pallets/ocex/src/lib.rs

```diff
@@ -836,15 +838,50 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Extrinsic to update ExchangeState
+		///This extrinsic will pause/resume the exchange according to flag
+		/// If flag is set to false it will stop the exchange
+		/// If flag is set to true it will resume the exchange
 		#[pallet::weight(1000000)]
 		pub fn set_exchange_state(origin: OriginFor<T>, state: bool) -> DispatchResult {
 			T::GovernanceOrigin::ensure_origin(origin)?;
 			<ExchangeState<T>>::put(state);
+
+			//SetExchangeState Ingress message store in queue
+			<IngressMessages<T>>::mutate(|ingress_messages| {
+				ingress_messages
+					.push(polkadex_primitives::ingress::IngressMessages::SetExchangeState(state))
```

Comment:
Push will panic, use try push

Reply:
ingress message is a normal vector, not a bounded vector so can't `try_push`. The argument am getting for this extrinsic is a bounded vector, so if anybody tries to pass a vector with higher than the limit we have decided it will fail.

Reply:
Added [test case here ](https://github.com/Polkadex-Substrate/Polkadex/blob/c4fff800b61ec692bdf2895da9f060fa10483f82/pallets/ocex/src/tests.rs#L1886) for sanity.